README.md

```diff
@@ -11,6 +11,7 @@ The project is currently alpha. While no breaking API changes are currently plan
  * [Usage](#usage)    
     * [Installation of the Operator](#install-the-operator)
     * [Deploy a sample Zookeeper Cluster](#deploy-a-sample-zookeeper-cluster)
+    * [Deploy a sample ZooKeeper Cluster with Ephemeral Storage](#Deploy-a-sample-zookeeper-cluster-with-ephemeral-storage)
```

Comment:
In case of ephemeral storage, what happens when a zk server is restarted? Does it come back up and join the ensemble as expected? Has this been tested. If there are concerns with DR when using ephemeral storage that should be made explicit.

Reply:
I have added documentation explicitly mentioning the concern with DR when using Ephemeral Storage.

Reply:
Ok. Looks better now.